frontend/src/components/footer.tsx

```diff
@@ -46,34 +51,72 @@ const Funders = () => {
+                                <Box gap>
+                                    <a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https://kinetic.openstax.org"><Icon icon={fbIcon} /></a>
+                                    <a target="_blank" href="https://www.instagram.com/openstax/"><Icon icon={igIcon} /></a>
+                                    <a target="_blank" href="https://twitter.com/intent/tweet?text=https://kinetic.openstax.org"><Icon icon={twIcon} /></a>
+                                </Box>
+                            </Col>
+                            <Col auto direction="column" gap>
+                                <div><img alt="Rice University logo" height="30" src={RiceLogoURL} /></div>
+                                <div><img alt="Open Stax logo" height="30" src={OpenStaxURL} /></div>
+                            </Col>
+                        </Row>
+                    </div>
+                </div>
+            </div>
+            :
+            <div className={cx('footer', 'mt-4', className)}>
+                {includeFunders && < Funders />}
+                <div css={{ backgroundColor: colors.darkBlue, color: 'white', a: { color: 'white' } }}>
+                    <div className='container-lg'>
+                        <Row className="py-2">
+                            <Row className="py-1">
+                                <HelpLink />
+                            </Row>
 
-                    </Row>
+                            <Row className="py-1">
+                                <ResourceLinks />
+                            </Row>
+
+                            <Row className="py-1">
+                                <b>Follow us</b>
+                                <Box gap>
+                                    <a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https://kinetic.openstax.org">
+                                        <Icon icon={fbIcon} height={30}/>
+                                    </a>
+                                    <a target="_blank" href="https://www.instagram.com/openstax/">
+                                        <Icon icon={igIcon} height={30}/>
+                                    </a>
+                                    <a target="_blank" href="https://twitter.com/intent/tweet?text=https://kinetic.openstax.org">
+                                        <Icon icon={twIcon} height={30}/>
+                                    </a>
+                                </Box>
+                            </Row>
+                            <Col className="py-1" direction="row" justify="around">
+                                <div><img alt="Rice University logo" height="30" src={RiceLogoURL} /></div>
+                                <div><img alt="Open Stax logo" height="30" src={OpenStaxURL} /></div>
+                            </Col>
+                        </Row>
+                    </div>
                 </div>
```

Comment:
yeah that's kinda hard to read 😁   How about extracting each side of the ternary out into separate components and then doing something like: 
```ts
export const Footer: React.FC<{ className?: string, includeFunders?: boolean }> = ({ className, includeFunders }) => (
  <div className={cx('footer', 'mt-4', className)}>
      {includeFunders && < Funders />}
      {useIsMobileDevice() ? <MobileFooter />  : <DesktopFooter />}
</div>
)
```